ci/cron/src/Docs.hs

```diff
@@ -220,18 +220,6 @@ sdkDocOpts = DocOptions
         proc_ ["tar", "xzf", "bazel-bin/docs/html.tar.gz", "--strip-components=1", "-C", temp </> show version]
   }
 
-damlOnSqlDocOpts :: DocOptions
```

Comment:
If you delete this, we’re not going to publish daml on sql docs for 1.18.2 or whatever other old releases we need. I think a cleaner solution would be to change `includedVersion` to stop at < 2.0.0-snapshot-whatever.